Loop/Managers/LoopDataManager.swift

```diff
@@ -1533,28 +1533,41 @@ extension LoopDataManager: DiagnosticReportGenerator {
 }
 
 extension LoopDataManager: LoopControl {
-    var automatedTreatmentState: LoopKit.AutomatedTreatmentState? {
+    private func neutralBasal(now: Date = Date()) -> Double? {
         guard let input = displayState.input else {
             return nil
         }
 
-        let now = Date()
-
-        let neutralBasal = input.basal.closestPrior(to: now)!.value
-        var scheduledBasalRate: Double
+        return input.basal.closestPrior(to: now)!.value
+    }
+    
+    func scheduledBasalRate(now: Date = Date()) -> Double? {
+        guard let neutralBasal = neutralBasal(now: now) else {
+            return nil
+        }
+        
         if let activeOverride = temporaryPresetsManager.presetHistory.activeOverride(at: now) {
-            scheduledBasalRate = neutralBasal / activeOverride.settings.effectiveInsulinNeedsScaleFactor
+            return neutralBasal / activeOverride.settings.effectiveInsulinNeedsScaleFactor
```

Comment:
"Neutral" basal is scheduled basal with overrides applied.  Is this "undoing? the override application, to get back to scheduled?